db/migrate/20200511014800_create_bios.rb

```diff
@@ -1,12 +0,0 @@
-class CreateBios < ActiveRecord::Migration[6.0]
```

Comment:
I don't think these migrations should be deleted since they track the history of the changes made to the database.

Reply:
Good catch! We have added the migrations back in. Let us know if you think any further actions need to be taken. Thanks